src/codeu/chat/util/store/StoreAccessor.java

```diff
@@ -1,3 +1,5 @@
+package codeu.chat.util.store;
```

Comment:
Did your intelli-j go nuts on this file or something? ;)

Sorta hilarious how confused github gets by the simple change in indenting, huh?